src/reference/enums.md

```diff
@@ -69,9 +73,9 @@ a textual description of the chosen value:
 🙌 can be used to compare whether two enum instances are equal:
 
 ```
-🆕⏰🥓❗️ ➡️ a
-🆕⏰🥓❗️ ➡️ b
+🆕⏰▶️🥓❗️ ➡️ a
+🆕⏰▶️🥓❗️ ➡️ b
 ↪️ a 🙌 b 🍇
-  😀 🔤Equal🔤❗️
+    😀 🔤Equal🔤❗️
```

Comment:
I need to change the indentation back (my editor keeps setting it to 4 spaces :facepalm: )